domains/celesta.json

```
@@ -0,0 +1,13 @@
+{
+  "description": "A miscellaneous webrepo, for containing files.",
+  "owner": {
+    "username": "3nf94Cmnds",
+    "email": "francescosuardi51@gmail.com",
+    "discord": "notveryc00l."
+  },
+  "record": {
+    "A": "37.101.98.55",
+    "TXT": "v=spf1 a ~all",
+    "NS": ["ns1.dynu.com", "ns11.dynu.com", "ns8.dynu.com"]
```

I think you only need NS records for this one
```suggestion
    "NS": ["ns1.dynu.com", "ns11.dynu.com", "ns8.dynu.com"]
```